packages/client/components/ScheduleDialog.tsx

```diff
@@ -0,0 +1,210 @@
+          videoType: gcalInput.videoType ?? undefined
+        }
+      : undefined
+    props.onStartActivity(gcalEventInput, hasRecurrence && rrule ? {rrule} : undefined)
+  }
+
+  const authGCal = () => {
+    if (!gcal?.cloudProvider) {
+      return
+    }
+    const {clientId, id: providerId} = gcal.cloudProvider
+    GcalClientManager.openOAuth(atmosphere, providerId, clientId, teamId, mutationProps)
+
+    SendClientSideEvent(atmosphere, 'Schedule meeting add gcal clicked', {
+      teamId: teamId,
+      service: 'gcal'
+    })
+  }
+
+  return (
+    <div className='space-y-4 p-4'>
+      <div className='text-lg font-semibold leading-none'>Schedule Your Meeting</div>
+      <div className='text-sm text-slate-800'>
+        Create a recurring meeting series or add the meeting to your calendar.
+      </div>
+      <div className='flex flex-col'>
+        <input
+          className='form-input rounded border border-solid border-slate-500 p-2 font-sans text-base hover:border-slate-600 focus:border-slate-600 focus:outline-none focus:ring-1 focus:ring-slate-600'
+          type='text'
+          name='title'
+          placeholder={placeholder}
+          value={title}
+          onChange={onNameChange}
+          min={1}
+          max={50}
+        />
+        {titleErr && <StyledError>{titleErr}</StyledError>}
+      </div>
+      {gcal?.cloudProvider &&
+        (gcal?.auth ? (
+          <Collapsible.Root
+            className='flex flex-col rounded border border-slate-500'
+            open={hasGcalEvent}
+            onOpenChange={onOpenGcalChange}
+          >
+            <Collapsible.Trigger className='flex cursor-pointer items-center bg-transparent p-2'>
+              <img src={gcalLogo} className='mr-2 h-6 w-6' />
+              <div className='grow text-left text-lg font-semibold leading-none'>
+                Calendar Invite
+              </div>
+              {hasGcalEvent ? <Close /> : <Add />}
+            </Collapsible.Trigger>
+            <Collapsible.Content className='space-y-4'>
+              <GcalSettings teamRef={team} settings={gcalInput} onSettingsChanged={setGcalInput} />
+            </Collapsible.Content>
+          </Collapsible.Root>
+        ) : (
+          <div>
+            <SecondaryButton className='h-11 pl-3 pr-4' onClick={authGCal}>
+              <img src={logo} className='mr-2' />
+              Connect to Google Calendar
+            </SecondaryButton>
+          </div>
+        ))}
+      {withRecurrence && (
+        <Collapsible.Root
+          className='flex flex-col rounded border border-slate-500'
+          open={hasRecurrence}
+          onOpenChange={onOpenRecurrenceChange}
+        >
+          <Collapsible.Trigger className='flex cursor-pointer items-center justify-between bg-transparent p-2'>
+            <EventRepeat className='mr-2 text-slate-600' />
+            <div className='grow text-left text-lg font-semibold leading-none'>Recurrence</div>
```

Comment:
+1 I think "Recurrence" makes sense to us, but some users might not understand what it means. Maybe we should copy GCal's wording and title it "Does not repeat"? If the user selects a recurrence setting, it says "Repeats every X days/months etc". 


![Screenshot 2024-03-08 at 15 16 11](https://github.com/ParabolInc/parabol/assets/39854876/d3ce7223-cad7-4525-8704-9575fb95079d)